src/core/router/Routes.js

```diff
@@ -4,6 +4,14 @@
 
 const ROOT :'/' = '/';
 
+const ACCESS_ID_PARAM :string = ':accessId';
+const ACCESS_REQUEST_PATH :string = `/request/${ACCESS_ID_PARAM}`;
```

Comment:
I would like to get rid of the `/request` part because the url looks like `/requests/request/{{id}}` and I think it's better if it's just `/requests/{{id}}`.

Reply:
i think the parent route should change to `/access` instead since there's likely going to be `/access/manage` down the road

Reply:
💯